### PR TITLE
Create a src and dist to separete transpiled code

### PR DESCRIPTION
**Pull request by raySavignone**
The code should be transpiled to es2015 for ease of use in most environments. A src directory stores all ES6 code for ease of development, then npm run build is used to populate a dist directory to use a the main in package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,11 @@
     "mssql": "4.0.4"
   },
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (ramanbuttar) [inline package.json]: can `"babel-core": "^6.26.0",` and `"babel-preset-es2015": "^6.24.1",` be moved out of the `dependencies` section as well?

File: src/objects/ProductionDbConfig.js [hunk 2]
```diff
@@ -43,15 +41,19 @@ class ProductionDbConfig {
...
     if (!config) {
       this.addConfigs(await this._fetchDatasources());
     }
-    
+
     return this.configs[database.toLowerCase()];
   }
+
+  toString () {
+    return this.dbConfig.toString();
```
Comment (ramanbuttar) [inline src/objects/ProductionDbConfig.js]: this doesn't seem like the right change. 
Both `StagingDbConfig` and `ProductionDbConfig` classes' `for_` methods return a `dbConfig` object which then needs to be outputted to `string`. This method will always return the `string` for this instance's `dbConfig` object, and not the one returned by the `for_` method. That's why the `toString` method needs to be on the `DbConfig` class. 

Maybe you can rename your variable when consuming all of this so it doesn't read so redundant.

File: src/objects/StagingDbConfig.js [hunk 3]
```diff
@@ -4,12 +4,16 @@ class StagingDbConfig {
...
-  
+
   async for_(database) {
     let config = this.dbConfig.copy();
     config.database = database.toLowerCase();
     return config;
   }
+
+  toString () {
+    return this.dbConfig.toString();
```
Comment (ramanbuttar) [inline src/objects/StagingDbConfig.js]: Same comment as above applies here.

**Review by ramanbuttar — CHANGES_REQUESTED** [review]
Like the change about splitting this into `src/` and `dist/` directories but one change to consider about the added method

File: package.json (hunk 1 above)
Comment (lucasklaassen) [reply package.json]: Let's ensure to remove the carrots `^` and lock down the versions to the ones we are using in our package-lock.json file.

**Review by ramanbuttar — APPROVED** [review]

**Comment by lucasklaassen** [comment]
Let's get some QA on this module if possible. Figure out a testing plan so we can prevent errors from going to production

**Pull request merged**